boards/shields/dfrobot_can_bus_v2_0/dfrobot_can_bus_v2_0.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_GPIO=y
+CONFIG_SPI=y
+CONFIG_CAN=y
```

Comment:
Should not be required as this is would be done on application side.
An application should enable CAN, either CAN controller this is a SoC IP or a device mounted on board or shield.
(And in short term, for application portability, other flags should be under if CAN condition, but we miss a piece in shield support to do that yet).

Reply:
I see, thanks for that, done

Reply:
@erwango is this issue now resolved?